daemon/delete.go

```diff
@@ -79,7 +79,12 @@ func (daemon *Daemon) rmLink(container *container.Container, name string) error
 func (daemon *Daemon) cleanupContainer(container *container.Container, forceRemove, removeVolume bool) (err error) {
 	if container.IsRunning() {
 		if !forceRemove {
-			err := fmt.Errorf("You cannot remove a running container %s. Stop the container before attempting removal or use -f", container.ID)
+			state := container.StateString()
+			procedure := "Stop the container before attempting removal or use -f"
```

Comment:
The `-f` / `--force` is really a CLI option. Wondering if the error message should be more generic, so that it makes sense for API clients as well. Should the "procedure" be printed by the client? not sure what we do with other error messages like this